Classes/Facet/FacetRendererFactory.php

```diff
@@ -22,7 +23,6 @@
 *  This copyright notice MUST APPEAR in all copies of the script!
 ***************************************************************/
 use TYPO3\CMS\Core\Utility\GeneralUtility;
-
```

Comment:
Can you please leave these empty lines? I actually like to have one empty line before the `use` statement and two after it. You don't have to add them everywhere, but please don't remove empty lines as they help with readability.